src/components/institution/OrganizationTitle.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, BoxProps, Typography } from '@mui/material';
+import { Organization } from '../../types/organization.types';
+import { getOrganizationHierarchy } from '../../utils/institutions-helpers';
+import { getLanguageString } from '../../utils/translation-helpers';
+
+interface OrganizationTreeProps extends Pick<BoxProps, 'sx'> {
+  organization: Organization;
+}
+
+export const OrganizationTitle = ({ organization }: OrganizationTreeProps) => {
```

Comment:
Jeg ville forventa at jeg bare får tilbake navnet på organization jeg sender inn som prop her, men man får faktisk navnet på alle enheter fra topp-nivå til den orgen det gjelder. Usikker på hva som er et godt navn, men på den vi har i dag er det `AffiliationHierarchy` for å indikere at du får tilbake hele hierarkiet, som også er tilfellet her

Reply:
Hva med OrganizationHierarchy?

Reply:
OrganizationHierarchyTitle / OrganizationHierarcyName ?